tutorial.md

```diff
@@ -197,11 +197,11 @@ This tutorial will use the latest version of `cni` from the master branch and bu
 Download the `cni` source tree:
```

Comment:
Does the 'cni' here and the 3 or four lines above need to be adjusted?  Or at least talk about plugins somewhere in this neighborhood?